### PR TITLE
add facebook files to test

### PR DESCRIPTION
**Pull request by sendilkumarn**
Followup PR based on [this](https://github.com/binast/binjs-ref/pull/69#issuecomment-373946283)

### CODE REVIEW

**Comment by Yoric** [comment]
Mmmmhhh, the test seems to timeout. That's a problem.

**Comment by Yoric** [comment]
I'm trying something in https://github.com/binast/binjs-ref/pull/78 that should help with the long-runnnig test_roundtrip.

**Comment by Yoric** [comment]
Ok, @sendilkumarn, could you try again now that #78 has landed?

**Comment by Yoric** [comment]
Drats, this now generates logs that exceed the limit imposed by Travis.

**Comment by Yoric** [comment]
Ok, this patch should now help 2f9b98300062a1916a9159acc6208db0c1e70e97 by limiting the amount of log we send.

**Comment by Yoric** [comment]
@sendilkumarn can you try rebasing this patch? I believe that this should now work.

**Comment by sendilkumarn** [comment]
now it has stack overflow

File: tests/test_roundtrip.rs [hunk 1]
```diff
@@ -45,87 +45,90 @@ fn test_roundtrip() {
...
-        let mut ast = binjs::specialized::es6::ast::Script::import(&ast)
-            .expect("Could not import AST");
-        binjs::specialized::es6::scopes::AnnotationVisitor::new()
-            .annotate_script(&mut ast);
-
+    let test_files = vec!["tests/data/spidermonkey/ecma_2/**/*.js", "tests/data/facebook/single/*.js"];
+    
+    for test_file in test_files {
+        let path = format!("{}/{}", env!("CARGO_MANIFEST_DIR"), test_file);
+        debug!(target: test_file, "Starting {} from {}", test_file, path);
```
Comment (Yoric) [inline tests/test_roundtrip.rs]: Oh, I hadn't seen that you had put `test_file` as `target`. Could you restore the debug messages that were there previously?

**Comment by Yoric** [comment]
Yeah, not good. We'll keep this PR suspended until we have found a way to make it avoid stack overflows.

File: tests/test_roundtrip.rs (hunk 1 above)
Comment (sendilkumarn) [reply tests/test_roundtrip.rs]: Poor me, I am on it.

**Comment by sendilkumarn** [comment]
Interesting, test fails...

**Comment by Yoric** [comment]
Just filed https://github.com/binast/binjs-ref/issues/91. If you wish to give a go at finding out where it stack overflows, be my guest :)

**Comment by Yoric** [comment]
Oh, wait, it's a different error now! I probably introduced a bug while working on https://github.com/binast/binjs-ref/issues/71 .

**Comment by sendilkumarn** [comment]
Yeah that is what I said. Interestimg

**Comment by Yoric** [comment]
Filed https://github.com/binast/binjs-ref/issues/92.

**Comment by Yoric** [comment]
So, I just found a little time to experiment: it fails during conversion from JSON to the typed AST.

**Comment by Yoric** [comment]
Ok, I managed to fix this as part of a larger PR: https://github.com/binast/binjs-ref/pull/97.

**Pull request closed without merging**